nbgrader/apps/api.py

```diff
@@ -48,7 +49,7 @@ def _log_level_changed(self, change):
             self.log_level = new
         self.log.setLevel(new)
 
-    def __init__(self, coursedir=None, authenticator=None, **kwargs):
+    def __init__(self, coursedir=None, authenticator=None, exchange=None, **kwargs):
```

Comment:
Needs an addition to the docstring explaining the new kwarg.

Reply:
Fixed this on my local branch.